src/test_lib_json/main.cpp

```diff
@@ -22,6 +22,12 @@
 #include <sstream>
 #include <string>
 
+#if __cplusplus >= 201103L || (defined(_CPPLIB_VER) && _CPPLIB_VER >= 520)
```

Comment:
Remove #if and assume a full C++11 implementation.

Reply:
But I think we might need to alter the AppVeyor tests in that case. Try it and see. If we have to drop support for older VisualStudio, that's fine, but we'll have to state that clearly in the release notes.